mpas_analysis/ocean/climatology_map_antarctic_melt.py

```diff
@@ -138,47 +137,128 @@ def __init__(self, config, mpasClimatologyTask, regionMasksTask,
+                # make a new subtask for this season and comparison grid
+                subtaskName = \
+                    f'plot_interface_melt_{season}_{comparisonGridName}'
+                subtask = PlotAntarcticMeltSubtask(
+                    self, season, comparisonGridName, remapClimatologySubtask,
+                    remapObservationsSubtask, controlConfig=controlConfig,
+                    subtaskName=subtaskName)
+
+                # In PlotAntarcticMeltSubtask, we will remove the obs from
+                # these plots if totalFluxVar is present so we only compare one
+                # field with obs
+
+                subtask.set_plot_info(
+                    outFileLabel='antMeltInterface',
+                    fieldNameInTitle='Melt Rate at Interface',
+                    mpasFieldName=mpasFieldName,
+                    refFieldName=refFieldName,
+                    refTitleLabel=refTitleLabel,
+                    diffTitleLabel=diffTitleLabel,
+                    unitsLabel=r'm a$^{-1}$ freshwater equiv.',
+                    imageCaption='Antarctic Melt Rate at Interface',
+                    galleryGroup='Melt Rate',
+                    groupSubtitle=None,
+                    groupLink='antarctic_melt_int',
+                    galleryName='Melt Rate at the Ice-ocean Interface')
+
+                self.add_subtask(subtask)
+
+        mpasFieldName = frazilFluxVar
+
+        if controlConfig is None:
+            refTitleLabel = None
+            refFieldName = None
+            diffTitleLabel = None
+
+        else:
+            controlRunName = controlConfig.get('runs', 'mainRunName')
+            refTitleLabel = f'Control: {controlRunName}'
+            refFieldName = mpasFieldName
+            diffTitleLabel = 'Main - Control'
+
+        for comparisonGridName in comparisonGridNames:
+            for season in seasons:
+                # make a new subtask for this season and comparison grid
+                subtaskName = \
+                    f'plot_interface_frazil_{season}_{comparisonGridName}'
+                subtask = PlotAntarcticMeltSubtask(
+                    self, season, comparisonGridName, remapClimatologySubtask,
+                    controlConfig=controlConfig, subtaskName=subtaskName)
+
+                subtask.set_plot_info(
+                    outFileLabel='antFrazil',
+                    fieldNameInTitle='Frazil Flux at Interface',
+                    mpasFieldName=mpasFieldName,
+                    refFieldName=refFieldName,
+                    refTitleLabel=refTitleLabel,
+                    diffTitleLabel=diffTitleLabel,
+                    unitsLabel=r'm a$^{-1}$ freshwater equiv.',
+                    imageCaption='Antarctic Frazil Flux',
+                    galleryGroup='Melt Rate',
+                    groupSubtitle=None,
+                    groupLink='antarctic_frazil_flux',
+                    galleryName='Frazil Flux through the Ice-ocean Interface')
```

Comment:
```suggestion
                    galleryName='Frazil accretion rate')
```
You can take or leave this suggestion. "through the interface" just sounded a little odd to me.

Reply:
```suggestion
                    galleryName='Frazil Flux through the Ice-ocean Interface, negative upwards')
```
I think we also need something like this, here or in caption, so the sign convention is clear.

Reply:
Done.